sec_certs/dataset.py

```diff
@@ -1010,32 +1028,49 @@ def color_check(current_key):
                            (self.certs[current_key].web_scan.module_name
                             if self.certs[current_key].web_scan.module_name else ''))
 
+        def get_processed_list():
+            if connection_list == "pdf":
+                cert_list = self.certs[key].pdf_scan.connections
+
+            elif connection_list == "web":
+                cert_list = self.certs[key].web_scan.connections
+
+            else:
+                cert_list = self.certs[key].processed.connections
+            return cert_list
+
         keys = 0
         edges = 0
 
         highlighted_vendor = 'Red Hat®, Inc.'
```

Comment:
Should this be here?

Reply:
Do you mean the `highlighted_vendor` part or the clumsy web/pdf/processed selection?

Reply:
I meant the hard-coded `highllighted_vendor` variable.

Reply:
I moved it to an argument with default value of `'Red Hat®, Inc.'`. I used this value because RH certificates populate a lot of space in the resulting graph and it's nice to see the results without an "extra touch".